robot_framework/process.py

```diff
@@ -45,11 +60,12 @@ def process(orchestrator_connection: OrchestratorConnection) -> None:
         request_type = _get_request_type_from_email(mail.body)
         # Send and delete email
         start_time = time.time()
-        return_data, rows_handled = handle_data(email_attachment, kombit_access, request_type, thread_count)
+        return_data, rows_handled = handle_data(email_attachment, kombit_access, request_type, process_arguments["thread_count"])
 
         orchestrator_connection.log_info(f"{rows_handled} Rows handled. Total time spent: {time.time()-start_time} seconds")
         _send_status_email(requester, return_data)
         graph_mail.delete_email(mail, graph_access)
+    os.remove(certificate_path)
```

Comment:
OpenOrchestrator removes all files in the current working dir, so this is not needed